dashboard/app/views/pd/application/teacher_application/new.html.haml

```diff
@@ -1,6 +1,12 @@
 = stylesheet_link_tag 'css/pd', media: 'all'
 %script{src: minifiable_asset_path('js/pd/application/teacher_application/new.js'), data: @script_data}
 
+:css
```

Comment:
Is this a change we should make for modals everywhere?

Reply:
never mind - i saw your commit message :)

Reply:
I don't think so right now.. I did some custom styling of this modal, just above, to make it look nice and consistent across pegasus & dashboard, but unfortunately I wasn't able to apply the custom styling beyond the levels that we explicitly render in this class.